spec/models/generic_file_spec.rb

```diff
@@ -1,7 +1,35 @@
 require 'rails_helper'
 
-RSpec.describe GenericFile do
-  it "should be able to persist" do
-    expect{GenericFile.new.save}.not_to raise_error
+RSpec.describe GenericFile do  
```

Comment:
Because the object that you are testing is not the GenericFile but the Enricher. If I were to look for the test that describes how the enricher works, I would expect to find it in the enricher_spec.rb, not the generic_file_spec.rb